http/http.ts

```diff
@@ -69,7 +70,8 @@ export class RequestContext {
      *
      */
     public getUrl(): string {
-        return this.url.toString();
+        const queryParams = "?" + queryString.default.stringify(this.url.query)
+        return this.url.origin + this.url.pathname + queryParams
```

Comment:
`url.toString()` is way more involved and does of lot stuff this code does not. 
you can simply supply the `stringify` function into `toString` and it will take that behaviour instead.

https://github.com/unshiftio/url-parse/blob/877225add9c3a2e33ef20ebef94dd397a0965666/index.js#L562

Reply:
Right this line has been removed and it is not being used at all, which is why I replaced it with the two green lines